src/exo/API_scheduling.py

```diff
@@ -1707,6 +1958,10 @@ def join_loops(proc, loop1_cursor, loop2_cursor):
     args:
         loop1_cursor     - cursor pointing to the first loop
         loop2_cursor     - cursor pointing to the second loop
+        rc               - (bool) whether to return relevant cursors
+
+    relevant cursors:
+        loop             - a cursor to the new joined loop
 
     rewrite:
         `for i in seq(lo, mid):`
```

Comment:
If every loop is 0-based on the bottom end, then this example code is bogus.  This is probably my own fault, but we should try to fix it to be accurate